### PR TITLE
update sofort icon

### PR DESCRIPTION
**Pull request by malcolm-mergulhao**
update existing Sofort icon.

### CODE REVIEW

**Comment by malcolm-mergulhao** [comment]
This changes the shade to a slightly different shade of pink (to match the Klarna Pay Later icon), and also rounds off the edges, which also matches with the Pay later icon better.

**Comment by malcolm-mergulhao** [comment]
@NathanPJF Would you be able to take another look please?

**Comment by matcaissy** [comment]
👋 is it possible to revert or address a fix? Looks like this added a regression in the themes: https://github.com/Shopify/themes/issues/190

File: app/assets/images/payment_icons/sofort.svg [hunk 1]
```diff
@@ -1 +1 @@
...
....022l-.388-.384c.318 1.553 1.702 2.593 3.32 2.593h.013zm-.09.516c-2.05-.037-3.773-1.547-3.828-3.69l-.471-.466c-.582-.584-.676-1.393-.208-1.873l.05-.053c.2-.207.464-.32.733-.334-.325-.641-.224-1.343.282-1.69l.072-.047c.55-.319 1.028-.196 1.437.21l-.47-3.865c-.158-.672.254-1.408.882-1.542a1.233 1.233 0 0 1 1.464.958l.801 3.42 1.015-3.476a1.285 1.285 0 0 1 1.565-.911c.617.183.989.811.854 1.436l-1.118 5.57.48.112c.14.02.322.103.494.287.313.334.464.886.396 1.692-.217 2.59-1.493 4.288-4.413 4.263l-.017-.001zm2.556-6.47L22.918 3.2a.711.711 0 0 0-.484-.832.767.767 0 0 0-.928.55l-1.292 4.427a.26.26 0 0 1-.433.108.258.258 0 0 1-.07-.124l-1.026-4.384a.713.713 0 0 0-.85-.56c-.334.07-.584.523-.483.926l.005.031.583 4.79.37.552c.16-.22.39-.358.664-.419a1.773 1.773 0 0 1 .824.024l2.006.468zm-3.726.51l-.572-.852a.258.258 0 0 1-.053-.079l-.262-.39c-.334-.486-.658-.654-1.04-.433l-.055.037c-.297.204-.338.698-.028 1.167l1.132 1.685a.651.651 0 0 0 .902.173l.056-.037a.64.64 0 0 0 .19-.205c-.282-.293-.35-.655-.27-1.067zm-.324 1.937a1.17 1.17 0 0 1-.983-.512l-.842-1.253a.587.587 0 0 0-.765.065l-.05.053c-.252.259-.196.75.2 1.147l1.44 1.427a.65.65 0 0 0 .915-.012l.05-.052a.65.65 0 0 0 .035-.863z"/></g></svg>
```
Comment (matcaissy) [inline app/assets/images/payment_icons/sofort.svg]: `<path fill="#FFF"` sets to white now 😬

**Comment by malcolm-mergulhao** [comment]
Hey @matcaissy I'll look in to fixing this and if it takes too long I'll revert this and the Klarna PR until I have fixed versions of the Icons. 
@FaycalSabaou can you send me a version of the Klarna and Sofort icons that fixes the issue here: https://github.com/Shopify/themes/issues/190

**Comment by FaycalSabaou** [comment]
@malcolm-mergulhao Not sure to understand what is the requested change from design side. can you tell me what need to be changed ?

**Comment by malcolm-mergulhao** [comment]
I think there were some workarounds in the issue i posted above, I don't really know much about SVGs though so I'm not sure how you would use them to fix the issue. Maybe someone in that thread would be able to help you.

**Comment by malcolm-mergulhao** [comment]
In slack discussion: I'm going to revert the change in core that bumped the gem version because these SVGs are fine, the display problems seem to be on core's end.

**Pull request merged**